convert2rhel/pkgmanager/__init__.py

```diff
@@ -100,3 +102,36 @@ def clean_yum_metadata():
         return
 
     loggerinst.info("Cached repositories metadata cleaned successfully.")
+
+
+@contextmanager
+def rpm_db_lock(pkg_obj):
+    """Context manager to handle rpm database termination.
+
+    .. note::
+        This context manager will only work with an instance of
+        `yum.rpmsack.RPMInstalledPackage`, as it will access specific
+        properties inside that class to close the RPM DB.
```

Comment:
Perhaps this `.. note::` is redundant with the `.. important::` admonition that comes after it?  If we want to keep it, maybe a slight rewording:
```suggestion
    .. note::
        This context manager will only do something with an instance of
        `yum.rpmsack.RPMInstalledPackage`, as it will access specific
        properties inside that class to close the RPM DB.  pkg_obj's from
        dnf are fine to pass in but will be a no-op as dnf manages the
        lifetime of the rpm_db lock fine on its own.
```